auth/token_gen.go

```diff
@@ -0,0 +1,63 @@
+package auth
```

Comment:
Don't shorten filenames when it isn't necessary :) I mean the `gen` part in `token_gen.go`.

Reply:
fixed